src/components/search-input/search-input.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import "./search-input.scss";
+import { FaSearch } from "react-icons/fa";
+
+export const SearchInput = ({
+  hasIcon = true,
+  onValueChange,
+  placeholder = "cauta",
+  value
+}) => {
+  const [currentValue, setCurrentValue] = useState(0);
+  return (
+    <div className={`__search-input-container ${hasIcon ? "__has_icon" : ""}`}>
+      <input
+        placeholder={placeholder}
+        value={value || currentValue || ""}
+        onChange={event => {
+          if (onValueChange) {
+            onValueChange(event);
+          }
+          if (!value) {
+            setCurrentValue(event.currentTarget.value);
```

Comment:
Should be fine for this use-case
```suggestion
            setCurrentValue(event.target.value);
```